tests/foreman/cli/test_contentaccess.py

```diff
@@ -92,40 +91,28 @@ def setUpClass(cls):
         )
         # Create repositories
         cls.repos = [
-            # Red Hat Enterprise Linux 7
```

Comment:
Since `repos` is one-element list, I guess it would be good to refactor it little more - rename to `repo` and change all callers to treat it as dict with data, not as list of dicts.

Reply:
I like keeping this as plural, as the called function is written for plural, and it allows for that use instead of only supporting singular.

Another consideration for future efforts @swadeley is actually using entity models in the tests and functions used here, instead of a dictionary to represent a repository.  Instead of key references and dictionaries defined within the tests, actually use a `Repository` class to model the entity, and create an instance of it here for the test to use and pass to other functions.

Reply:
@mirekdlugosz , the `setup_cdn_and_custom_repositories` function wants a "a list of dict repositories options", so I think I cannot just change `cls.repos = [ {` to `cls.repos = {` with just one dict.
Re-factor to use simpler `setup_org_for_a_rh_repo` is an option.

What do you think?

Thank you

Reply:
@swadeley I was thinking about internal uses in test here. You can always wrap in list when passing to helper.
Although it seems that this helper is used only handful of times in robottelo, and in other instances it gets one-element lists.
Using simpler helper sounds like a good approach, too.

Reply:
Thanks for suggestions

BTW I posted my comment above without having seen Mike's comment (as I had the this open and did not reload).